api/controllers/v0.3/DependentAccountAPI.js

```diff
@@ -52,21 +71,34 @@ const DependentAccountAPI = (args) => {
       const canCreateDependents = checkDependentLimit(patron.varFields);
 
       if (!canCreateDependents) {
-        response["eligible"] = false;
-        response["description"] =
-          "This patron has reached the limit to create dependent accounts.";
+        throw new NotEligibleCard(
+          "You have reached the limit of dependent cards you can receive via online application."
```

Comment:
for this business logic error and the ones below related to `NotEligibleCard`, are you still returning a response with `eligible` and `description` fields as we had before?

Reply:
and do these business logic errors still end up with a 200 status code?

Reply:
No, those errors are now being returned as a 400 response. So now you'll get a response like:

```
{
    status: 400,
    type: 'not-eligible-card',
    message: '....'
}
```

Check this PR's description for more examples.